README.md

```diff
@@ -1,110 +1,97 @@
-+ [git](https://git-scm.com/)
+![sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/large--large_pop_brute_force.png](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/large--large_pop_brute_force.png)
 
-### Building FLAME GPU 2
+### Large Populations (Spatial)
 
-FLAME GPU 2 uses [CMake](https://cmake.org/), as a cross-platform process, for configuring and generating build directives, e.g. `Makefile` or `.vcxproj`. This is used to build the FLAMEGPU2 library, examples, tests and documentation.
+This figure shows how the average simulation time scales with ensemble size for different large population sizes of a spatial messaging Boids model.
+	+ Population sizes are stepped between 2048 and 8192 at intervals of 2048
+	+ Simulation timing is measured over 500 steps
 
-Below the core commands are provided, for the full guide refer to the main [FLAMEGPU2 guide](https://github.com/FLAMEGPU/FLAMEGPU2/blob/master/README.md).
+![sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/large--large_pop.png](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/large--large_pop.png)
 
-#### Linux
+## Building and Running the Benchmark
 
-Under Linux, `cmake` can be used to generate makefiles specific to your system:
+Detail of dependencies and the `cmake` build process are described in full in the [FLAMEGPU2-example-template Repo](https://github.com/FLAMEGPU/FLAMEGPU2-example-template) and are not repeated here. The benchmark should be built with seatbelts off (e.g. `-DSEATBELTS=OFF` passed to the `cmake` configuration step) to disable additional run-time checks and optionally disabling Python Swig support which is not needed for this model(-DBUILD_SWIG_PYTHON=OFF ). E.g. for Volta (`SM_70`) GPUs under Linux.
 
 ```bash
-mkdir -p build && cd build
-cmake .. 
-make -j8
+# Configure 
+cmake . -B build -DCMAKE_BUILD_TYPE=Release -DSEATBELTS=OFF -DBUILD_SWIG_PYTHON=OFF -DCUDA_ARCH=70
+cmake --build build -j`nproc` 
+make
 ```
 
-The option `-j8` enables parallel compilation using upto 8 threads, this is recommended to improve build times.
-
-By default a `Makefile` for the `Release` build configuration will be generated. This is the configuration which was used to generate the results for the paper/talk.
-
-#### Windows
-
-*Note: If installing CMake on Windows ensure CMake is added to the system path, allowing `cmake` to be used via `cmd`, this option is disabled within the installer by default.*
+### Execution and Data generation
 
-When generating Visual studio project files, using `cmake` (or `cmake-gui`), the platform **must** be specified as `x64`.
-
-Using `cmake` this takes the form `-A x64`:
+The `FLAMEGPU2_INC_DIR` environment variable must be set for run-time compilation (TRC) to succeed. This can be done at the same time as execution of the model. E.g. 
 
 ```bash
-mkdir build && cd build
-cmake .. -A x64
-ALL_BUILD.sln
+cd build
+FLAMEGPU2_INC_DIR=./_deps/flamegpu2-src/include/ ./bin/Release/ensemble-benchmark 
```

Comment:
I do not believe this is required (usually)

Reply:
fixed